src/Components/Components/src/Microsoft.AspNetCore.Components.csproj

```diff
@@ -9,10 +9,14 @@
     <RazorLangVersion>3.0</RazorLangVersion>
   </PropertyGroup>
 
-    <ItemGroup>
+  <ItemGroup>
     <Compile Include="..\..\Shared\src\JsonSerializerOptionsProvider.cs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\Analyzers\src\Microsoft.AspNetCore.Components.Analyzers.csproj" />
+  </ItemGroup>
```

Comment:
A few questions

- Does our build system actually allow this? I thought it was prohibited?
- Does this reference the `.Analyzers` assembly as a dll? We don't want this showing up as a real reference.

Reply:
> * Does our build system actually allow this? I thought it was prohibited?

We'll see if the build system allows it. I could dotnet build it locally but I'm unable to run a full build so we'll let the PR tell us that.

> * Does this reference the `.Analyzers` assembly as a dll? We don't want this showing up as a real reference.

Oops I meant to include that in the issue commit / PR message. It does not, I had the same worry so I made sure to verify that specifically 😄

Reply:
How does that work tho? I would expect you to need `ReferenceOutputAssembly=false`

Reply:
> How does that work tho? I would expect you to need `ReferenceOutputAssembly=false`

Totally agree, I did that at first and that ended up nuking the package reference in the generated nupkg. Played around with it a bit more and found that Analyzers must be special cased at some layer to not require `ReferenceOutputAssembly=false`. Unless ILSpy is lying to me, it's not required:

![image](https://user-images.githubusercontent.com/2008729/60931224-9d8e6280-a26d-11e9-9fe5-bccd6fc24058.png)

Reply:
Recall our conversation with the Roslyn team a few weeks ago... The set of "assembly references" input to the compiler is not the same as the set of "assembly references" output.

This is only happening by coincidence because we don't reference any types from the analyzers assembly.

This really needs to be done properly, sorry.